src/controllers/update/statement.bo.cease.controller.ts

```diff
@@ -0,0 +1,7 @@
+import { NextFunction, Request, Response } from "express";
+import { UPDATE_STATEMENT_BO_CEASE_PAGE, UPDATE_STATEMENT_BO_CEASE_URL } from "../../config";
+import { getStatmentBOCease } from "../../utils/update/statement.bo.cease";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  return getStatmentBOCease(req, res, next, UPDATE_STATEMENT_BO_CEASE_PAGE, UPDATE_STATEMENT_BO_CEASE_URL);
+};
```

Comment:
Is it worth adding in a dummy post controller even if we don't know where this page will be redirecting to just yet?

Reply:
you should not implement something that is not in requirements on the ticket. You don't know what the future requirements are yet. You just follow and implement what is in the ticket requirements and not things that aren't there.